feat(wbh): docs content review (#4428)
scaleway/docs-content: pages/instances/concepts.mdx

## Carrier-grade NAT (CGNAT)

> Scaleway used a 1:1 carrier-grade (CGNAT) setup for legacy Instance types. With this architecture, each Instance is assigned a unique NAT IP that directly maps to a unique public IP, unlike standard CGNAT, where multiple users share a single public IP.

Reviewer: ```suggestion
Scaleway used a 1:1 carrier-grade (CGNAT) setup for legacy Instance types. With this architecture, each Instance was assigned a unique NAT IP that directly mapped to a unique public IP, unlike standard CGNAT, where multiple users share a single public IP.
```

> This setup ensured that all outbound and inbound traffic for an Instance is translated between these two addresses, preserving the exclusivity of IP addresses and avoiding certain connectivity issues related to a shared IP setup.

Reviewer: ```suggestion
This setup ensured that all outbound and inbound traffic for an Instance was translated between these two addresses, preserving the exclusivity of IP addresses and avoiding certain connectivity issues related to a shared IP setup.
```
